src/platform/cc13x2_26x2/CC13X2_26X2Config.cpp

```diff
@@ -389,43 +417,49 @@ CHIP_ERROR CC13X2_26X2Config::FactoryResetConfig(void)
     NVINTF_nvProxy_t nvProxy = { 0 };
     uint8_t status           = NVINTF_SUCCESS;
 
+    // Delete items with the config, kvs_key and kvs_value itemIDs. Items with the factory
+    // and counter itemIDs are not deleted.
```

Comment:
The counters, or at least some of them, should also be deleted. I looks like other platforms are also missing that, but that is a separate issue.

`The RebootCount attribute SHALL only be reset to 0 upon a factory reset of the Node.`
`The TotalOperationalHours attribute SHALL only be reset upon a factory reset of the Node.`

I could not find any statement in the spec that the LifeTimeCounter should also be reset in a factory reset 🤔 so not sure what is best here, maybe move it back to factory, even though that is not really where it belongs.

Reply:
I moved the LifeTimeCounter back to Factory and added the lines needed to delete the Counters keys.